packages/vue-app/template/router.scrollBehavior.js

```diff
@@ -31,9 +31,9 @@ export default function (to, from, savedPosition) {
   // Scroll to the top of the page if...
   if (
       // One of the children set `scrollToTop`
-      (Pages.some(Page => Page.options.scrollToTop) ||
+      (Pages.some(Page => Page.options && Page.options.scrollToTop) ||
```

Comment:
Could we use this?
```suggestion
      (Pages.some(Page => Page.options?.scrollToTop) ||
```

Reply:
This may add extra polyfill if user is not using optional chaining.

Reply:
Let's merge this and I may make some refactoring to this file,